buildSrc/subprojects/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -184,8 +186,35 @@ class PerformanceTestPlugin : Plugin<Project> {
             // Never up-to-date since it reads data from the database.
             outputs.upToDateWhen { false }
         }
+
+        val performanceScenarioJson = rootProject.file(".teamcity/performance-tests-ci.json")
+        val tmpPerformanceScenarioJson = rootProject.buildDir.resolve("performance-tests-ci.json")
```

Comment:
How about we resolve the tmp file in the current project directory?

Reply:
Makes sense. Done.